r2d3_en.rst

```diff
@@ -0,0 +1,298 @@
+       expert_data[i]['is_expert'] = [1] * expert_cfg.policy.collect.unroll_len
+   ...
+   # If it is an agent experience sample, this key value is 0
+   for i in range(len(new_data)):
+       new_data[i]['is_expert'] = [0] * expert_cfg.policy.collect.unroll_len
+4. Pre-training. Before the agent interacts with the environment, we can use the expert demo samples to pre-train the Q network, hoping to get a good initialization parameter to speed up the subsequent training process.
+
+.. code::
+
+    for _ in range(cfg.policy.learn.per_train_iter_k):  # pretrain
+        if evaluator.should_eval(learner.train_iter):
+            stop, reward = evaluator.eval(learner.save_checkpoint, learner.train_iter, collector.envstep)
+            if stop:
+                break
+        # Learn policy from collected demo data
+        # Expert_learner will train ``update_per_collect == 1`` times in one iteration.
+        train_data = expert_buffer.sample(learner.policy.get_attribute('batch_size'), learner.train_iter)
+        learner.train(train_data, collector.envstep)
+        if learner.policy.get_attribute('priority'):
+            expert_buffer.update(learner.priority_info)
+
+Implementations
+====
+
+of r2d3's policy ``R2D3Policy`` is defined as follows:
+
+.. autoclass:: ding.policy.r2d3.R2D3Policy
+   :members: _forward_learn
+   :noindex:
+
+of dqfd's loss function ``nstep_td_error_with_rescale`` is defined as follows:
+
+.. autofunction:: ding.rl_utils.td.dqfd_nstep_td_error_with_rescale
+   :noindex:
+
+.. note::
+    The input of the network in our current r2d3 policy implementation is only the state observation at time t, not including the action and reward at time t-1, nor the extra information vector :math:`f_{t}` .
+
+
+Benchmark Algorithm Performance
+===========
+
+We conducted a series of comparative experiments in the PongNoFrameskip-v4 environment to verify: 1. The proportion of expert samples in a mini-batch used for training pho, 2. The proportion of expert demonstrations, 3. Whether to use pre-training The effect of different parameter settings such as l2 regularization on the final performance of the r2d3 algorithm.
+
+.. note::
+Our expert data is generated via `ppo_offpolicy_collect_traj.py <https://github.com/opendilab/DI-engine/blob/main/ding/policy/ppo_offpolicy_collect_traj.py>`_ ,
+Its expert model comes from the expert model obtained after the r2d2 algorithm is trained to converge on this environment. All experiments below seed=0.
+
+The r2d2 baseline algorithm setting is recorded as r2d2_n5_bs2_ul40_upc8_tut0.001_ed1e5_rbs1e5_bs64, where:
+
+- n means nstep,
+- bs for burnin_step,
+- ul means unroll_len,
+- upc means update_per_collect,
+- tut means target_update_theta,
+- ed means eps_decay,
+- rbs means replay_buffer_size,
+- bs means batch_size,
+
+See `r2d2 pong config <https://github.com/opendilab/DI-engine/blob/main/dizoo/atari/config/serial/pong/pong_r2d2_config.py>`_ for details .
+
+..
```

Comment:
this table is in wrong format, can you copy from the original chinese `r2d3_zh.rst` to correct it